### DISCUSSION TITLE
Problems with showdialog on child/main window

### DISCUSSION THREAD

**Discussion by kondareddy** [MahApps/MahApps.Metro#4145] [Q&A]
We have a scenario to have child window for functionality apart from main window. When we do that any dialog opened on main window then we can't access child window at all and it's vice versa..

![image](https://user-images.githubusercontent.com/1445374/124100051-bfe1a200-da7b-11eb-9dda-a6e3e570b1fb.png)

Any idea how this is happening? user should be able perform actions on both windows.

FYI - No owner set to child window as well.

**Comment by punker76** [comment]
@kondareddy If you use `ShowDialog` then the child window will be modal. If you don't want this then use the `Show` method.
